Testing/functional/e2e/cypress/integration/timeline/filter.js

```diff
@@ -63,11 +63,11 @@ describe("Filters", () => {
         cy.get("[data-testid=filterStartDateInput] input")
             .focus()
             .clear()
-            .type("2020-06-14");
+            .type("2020-JUN-14");
         cy.get("[data-testid=filterEndDateInput] input")
             .focus()
             .clear()
-            .type("2020-06-14");
+            .type("2020-JUN-14").focus();
```

Comment:
For consistency, should the month be in title case ("Oct" and "Jun")?

Reply:
Leo explained that we have a mask in place that capitalizes these particular inputs.